packages/lime-librenet6/files/etc/tinc/librenet6/tinc-up

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+ifconfig $INTERFACE up txqueuelen 1000
```

Comment:
ifconfig is deprecated! Isn't ip installed by default on openwrt at least some minimal version from busybox ?